api/src/main/java/jakarta/data/Limit.java

```diff
@@ -39,14 +40,13 @@
  * secondMostExpensive50 = products.findByNameLike(pattern, Limit.range(51, 100), Sort.desc("price"));
  * </pre>
  *
- * <p>A repository method will fail if</p>
+ * <p>A repository method may not be declared with:
  * <ul>
- * <li>multiple {@code Limit} parameters are supplied to the
- *     same method.</li>
- * <li>{@code Limit} and {@link PageRequest} parameters are supplied to the
- *     same method.</li>
- * <li>a {@code Limit} parameter is supplied in combination
- *     with the {@code First} keyword.</li>
+ * <li>more than one parameter of type {@code Limit},</li>
+ * <li>a parameter of type {@code Limit} and a parameter of type
+ *     {@link PageRequest}</li>, or
```

Comment:
Needed for the CI to pass. Error is:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.6.0:jar (attach-javadocs) on project jakarta.data-api: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1
Error:  /home/runner/work/data/data/api/src/main/java/jakarta/data/Limit.java:47: error: text not allowed in <ul> element
Error:   *     {@link PageRequest}</li>, or
Error:                                 ^
Error: [ERROR] 1 error
```

```suggestion
 * <li>a parameter of type {@code Limit} and a parameter of type
 *     {@link PageRequest}, or</li>
```